Apps/Playground/package.json

```diff
@@ -14,8 +14,8 @@
     "iosCmake": "node scripts/tools.js iosCMake"
   },
   "dependencies": {
-    "@babylonjs/core": "^5.0.0-alpha.30",
-    "@babylonjs/loaders": "^5.0.0-alpha.30",
+    "@babylonjs/core": "^5.0.0-alpha.34",
```

Comment:
BabylonNative needs to be updated to this package before we update here.

Reply:
not sure why this wasn't already done: https://github.com/BabylonJS/BabylonNative/blob/master/Apps/package.json

Reply:
I didn't update BabylonNative because it's not a required change (and arguably didn't need to update it for the BRN package itself, just the test app), but I don't mind updating it in BN as well.